1_Mechanics/03_CharacterCreation.md

```diff
@@ -1,5 +1,17 @@
 # Character Creation
 
+## Create Your Dealer
```

Comment:
How much of this section is duplicative with the player guide full?

Reply:
I just felt like this doc needed an 'intro' section that was specific to DofA terminology before we dive right into Character Backgrounds. We'll review and do a full edit of this doc in a future PR.